packages/multichain/src/scope/validation.test.ts

```diff
@@ -0,0 +1,185 @@
+import type { ExternalScopeObject } from './types';
+import { isValidScope, validateScopes } from './validation';
+
+const validScopeString = 'eip155:1';
+const validScopeObject: ExternalScopeObject = {
+  methods: [],
+  notifications: [],
+};
+
+describe('Scope Validation', () => {
+  describe('isValidScope', () => {
+    it.each([
```

Comment:
Similar comment as elsewhere. I am finding this a bit difficult to read in its current form. I end up having to go back and forth between the data that's used to construct the test and the actual body of the test to understand what each of these values means and how they're used. What are your thoughts on writing these tests out?